.gitignore

```diff
@@ -47,6 +47,9 @@ mobile/Preview.html
 # Coverage
 coverage/
 
+# transifex
+.transifexrc
```

Comment:
If you think that can be useful for other developers you can push this change in a cozy-template PR :)